client/src/components/common/names-capture.vue

```diff
@@ -140,8 +141,9 @@
           </span>
         </v-col>
       </v-row>
+
       <v-row class="mt-5" v-if="showSecondAndThirdNameChoices">
-        <v-col cols="2" class="py-0 label-style" align-self="start" key="static-2">
+        <v-col cols="2" class="label-style align-self-start pt-0" key="static-2">
```

Comment:
What this (and the similar change above, below and on other pages) means is that the left-side label is aligned to the top of the right-side text inputs. It was this way on some pages already but now it's consistent.